kolibri/core/tasks/storage.py

```diff
@@ -291,9 +292,8 @@ def test_table_readable(self):
         """
         Do a quick query to raise errors if the database is unusable.
         """
-        with self.session_scope() as s:
-            q = s.query(ORMJob)
-
+        with self.engine.connect() as conn:
+            q = conn.execute(select(ORMJob))
             q.first()
 
     def count_all_jobs(self, queue=None):
```

Comment:
Sure, I did not update this function before because we were modifying the query based on the condition. But I have found something that can be used to produce similar behavior with connection-based queries.
https://docs.sqlalchemy.org/en/20/core/connections.html#synopsis
Will push a new commit.
Thanks for the review.

Reply:
@vkWeb I have checked the function `count_all_jobs` is not being used anywhere in the code. Is there any need for this function or should we remove it?

Reply:
Yes let's remove it.